helm/examples/efs/trust-policy.json.template

```diff
@@ -0,0 +1,18 @@
+{
+  "Version": "2012-10-17",
```

Comment:
is this template (and helm/examples/efs/iam-policy-example.json) copied from somewhere?
Does it need a reference/acknolegement comment? 
Or is the Version = 2012-10-17 correct , if it is now versioning?

Reply:
Copied from AWS. It is always " Version = 2012-10-17 "

Reply:
ok